pkg/apis/cr/v1alpha1/types.go

```diff
@@ -109,9 +112,14 @@ type ActionSpec struct {
 
 // ActionSetStatus is the status for the actionset. This should only be updated by the controller.
 type ActionSetStatus struct {
-	State    State          `json:"state"`
-	Actions  []ActionStatus `json:"actions,omitempty"`
-	Error    Error          `json:"error,omitempty"`
+	// State represents the current state of the ActionSet.
+	State State `json:"state"`
```

Comment:
Let's also mention all the possible values for State here. Let me know if you need some help.

Reply:
Are the state values: Pending, Running, Failed and complete?

Reply:
yes